pio/st7789_lcd/st7789_lcd.c

```diff
@@ -90,7 +92,7 @@ int main() {
     PIO pio = pio0;
     uint sm = 0;
     uint offset = pio_add_program(pio, &st7789_lcd_program);
-    st7789_lcd_program_init(pio, sm, offset, PIN_DIN, PIN_CLK, SERIAL_CLK_DIV);
+	st7789_lcd_program_init(pio, sm, offset, SPI_POLARITY, PIN_DIN, PIN_CLK, SERIAL_CLK_DIV);
```

Comment:
Might be tabs vs 4 spaces?

Reply:
I edited via the web page.  Should I fix this?